From 9035899c32cd7f753a460565ac6c69b20c2b0b7f Mon Sep 17 00:00:00 2001
From: Clement Lefebvre <clement.lefebvre@linuxmint.com>
Date: Tue, 8 Nov 2016 19:22:49 +0000
Subject: [PATCH] Layout/l10n: Fix minor visual issues

- The text in the entry didn't fit in some languages
- In the absence of flags, the text in the entry was stuck to the left border
- Some msgids could be improved
---
 cinnamon-screensaver.pot             | 50 ++++++++++++++++++------------------
 src/cinnamon-screensaver-gtk3.18.css |  4 +--
 src/passwordEntry.py                 |  4 ++-
 src/unlock.py                        |  4 +--
 4 files changed, 32 insertions(+), 30 deletions(-)

diff --git a/cinnamon-screensaver.pot b/cinnamon-screensaver.pot
index b21f9c3..b5cf978 100644
--- a/cinnamon-screensaver.pot
+++ b/cinnamon-screensaver.pot
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2016-09-23 22:51-0400\n"
+"POT-Creation-Date: 2016-11-08 19:22+0000\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
@@ -18,85 +18,85 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
 
-#: src/cinnamon-screensaver-command.py:34
+#: src/cinnamon-screensaver-command.py:37
 msgid "Causes the screensaver to exit gracefully"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:36
+#: src/cinnamon-screensaver-command.py:39
 msgid "Query the state of the screensaver"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:38
+#: src/cinnamon-screensaver-command.py:41
 msgid "Query the length of time the screensaver has been active"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:40
+#: src/cinnamon-screensaver-command.py:43
 msgid "Tells the running screensaver process to lock the screen immediately"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:42
+#: src/cinnamon-screensaver-command.py:45
 msgid "Turn the screensaver on (blank the screen)"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:44
+#: src/cinnamon-screensaver-command.py:47
 msgid "If the screensaver is active then deactivate it (un-blank the screen)"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:46
+#: src/cinnamon-screensaver-command.py:49
 msgid "Version of this application"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:48
+#: src/cinnamon-screensaver-command.py:51
 msgid "Message to be displayed in lock screen"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:77
+#: src/cinnamon-screensaver-command.py:80
 msgid "The screensaver is active\n"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:79
+#: src/cinnamon-screensaver-command.py:82
 msgid "The screensaver is inactive\n"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:83
+#: src/cinnamon-screensaver-command.py:86
 msgid "The screensaver is not currently active.\n"
 msgstr ""
 
-#: src/cinnamon-screensaver-command.py:85
+#: src/cinnamon-screensaver-command.py:88
 #, python-format
 msgid "The screensaver has been active for %d second.\n"
 msgid_plural "The screensaver has been active for %d seconds.\n"
 msgstr[0] ""
 msgstr[1] ""
 
-#: src/clock.py:59
+#: src/clock.py:60
 msgid "%A, %B %e"
 msgstr ""
 
-#: src/playerControl.py:158
+#: src/passwordEntry.py:24 src/unlock.py:278
+msgid "Please enter your password..."
+msgstr ""
+
+#: src/playerControl.py:168
 msgid "--:--"
 msgstr ""
 
-#: src/playerControl.py:161
+#: src/playerControl.py:171
 msgid "%M:%S"
 msgstr ""
 
-#: src/playerControl.py:163
+#: src/playerControl.py:173
 msgid "%H:%M:%S"
 msgstr ""
 
-#: src/unlock.py:40 src/unlock.py:295
-msgid "Enter password..."
-msgstr ""
-
-#: src/unlock.py:212
+#: src/unlock.py:172
 msgid "You have the Caps Lock key on."
 msgstr ""
 
-#: src/unlock.py:244
+#: src/unlock.py:218
 msgid "Checking..."
 msgstr ""
 
-#: src/unlock.py:296
-msgid "Password incorrect - try again."
+#: src/unlock.py:279
+msgid "Incorrect password"
 msgstr ""
diff --git a/src/cinnamon-screensaver-gtk3.18.css b/src/cinnamon-screensaver-gtk3.18.css
index 874a5d5..e70cbe8 100644
--- a/src/cinnamon-screensaver-gtk3.18.css
+++ b/src/cinnamon-screensaver-gtk3.18.css
@@ -96,7 +96,7 @@
     background-image: none;
     background-color: transparent;
     border-width: 1px;
-    padding: 5px 10px 5px 0px;
+    padding: 5px 10px 5px 5px;
 }
 
 .csstage .passwordentry:focused {
@@ -126,7 +126,7 @@
 }
 
 .csstage .passwordentry.image.left {
-    padding: 0 20px 0 5px;
+    padding: 0 15px 0 5px;
 }
 
 .csstage .transparentbutton {
diff --git a/src/passwordEntry.py b/src/passwordEntry.py
index a2ff0ff..781d4b2 100644
--- a/src/passwordEntry.py
+++ b/src/passwordEntry.py
@@ -21,11 +21,13 @@ def __init__(self):
         self.get_style_context().add_class("passwordentry")
 
         self.set_halign(Gtk.Align.FILL)
+        placeholder_text = _("Please enter your password...")
+        self.set_width_chars(len(placeholder_text) + 3) # account for the flag
         self.set_has_frame(True)
         self.set_input_purpose(Gtk.InputPurpose.PASSWORD)
         self.set_visibility(False)
         self.set_property("caps-lock-warning", False)
-        self.set_placeholder_text (_("Enter password..."))
+        self.set_placeholder_text (placeholder_text)
         self.set_can_default(True)
 
         self.current_icon_name = None
diff --git a/src/unlock.py b/src/unlock.py
index db2dc77..6e22d9a 100644
--- a/src/unlock.py
+++ b/src/unlock.py
@@ -275,8 +275,8 @@ def authentication_failed(self):
         """
         self.clear_entry()
 
-        self.password_entry.set_placeholder_text (_("Enter password..."))
-        self.auth_message_label.set_text(_("Password incorrect - try again."))
+        self.password_entry.set_placeholder_text (_("Please enter your password..."))
+        self.auth_message_label.set_text(_("Incorrect password"))
 
         self.password_entry.grab_focus()
 
